src/www/components/popup/TeamPopup.jsx

```diff
@@ -70,15 +93,25 @@ class TeamPopup extends Component {
   handleSubmit = (event) => {
     event.preventDefault();
     const { teamName, description, teamCategories, teamPicture } = this.state;
-
     if (teamName !== "" && description !== "") {
       this.setState({
         teamNameHighlight: false,
       });
       this.createTeam(teamName, description, teamCategories, teamPicture);
+    } else {
```

Comment:
else := `teamName === "" || description === ""`
This isn't correct since otherwise if both of them aren't empty, the other branch will be executed leading to a team creation and not an update!